src/FlaUI.WebDriver.UITests/SessionTests.cs

```diff
@@ -315,6 +315,35 @@ public void NewCommandTimeout_NotExpired_DoesNotEndSession()
             Assert.That(() => driver.Title, Throws.Nothing);
         }
 
```

Comment:
You only added tests for NewCommandTimeout, how about adding asserts to the `NewSession_AppTopLevelWindowTitleMatch_IsSupported` and `NewSession_AppTopLevelWindow_IsSupported` tests above that close the sessions immediately? In that case the processes should also be left open.

Reply:
Do you mean to change the `using var driver` statement to a `using (var driver = ...)` statement, and outside the block check that the process is still running?

(Feel free to make this change yourself and push to the branch if that'd be easier than explaining it ;) )

Reply:
Yes, that is what I meant - or call the `driver.Quit()` explicitly, which makes the test easier to read.

Reply:
Ah yes `driver.Quit()` is more readable. Done!